lib/graphql/persisted_queries/schema_patch.rb

```diff
@@ -41,7 +44,12 @@ def verify_http_method=(verify)
         end
       end
 
+      def persisted_queries_tracing_enabled?
+        @persisted_queries_tracing_enabled
+      end
+
       def multiplex(queries, **kwargs)
+        persisted_query_store.tracers = tracers if persisted_queries_tracing_enabled?
```

Comment:
This is a little awkward, but something like this is necessary since we don't know if tracers will be configured before or after our plugin is configured (it is dependent on how someone sets up their schema).  An alternative would be to patch `GraphQL::Schema.tracer` to pass through to also set the tracer on the store.

Reply:
I ended up moving this to a patch on `GraphQL::Schema.tracer` and it's a bit clearer now.